chromadb/test/db/test_system.py

```diff
@@ -728,14 +728,6 @@ def test_update_segment(sysdb: SysDB) -> None:
     result[0]["collection"] = segment["collection"]
     assert result == [segment]
 
-    # Update collection to new value
```

Comment:
should we add a test for update a segment with the wrong colleciton to enforce that contract